sql/001-create-table-and-load-data.sql

```diff
@@ -8,6 +8,6 @@ CREATE TABLE fishes (
   PRIMARY KEY(id)
 );
 
-INSERT INTO fishes (name,weight,price) VALUES ("タイ", "2200g","￥1400");
-INSERT INTO fishes (name,weight,price) VALUES ("カニ", "500g","￥10000");
-INSERT INTO fishes (name,weight,price) VALUES ("マグロ", "30000g","￥93510");
+INSERT INTO fishes (name,weight,price) VALUES ("タイ", "1000g","￥1036");
```

Comment:
重さ、値段はINTで保持するほうがいいですね。
```suggestion
INSERT INTO fishes (name,weight,price) VALUES ("タイ", "1000","1036");
```
重さの単位も明示したいならカラム名に情報をもたせるのもいいでしょう。
```suggestion
INSERT INTO fishes (name,weightInGram,price) VALUES ("タイ", "1000","1036");
```
もし通貨情報も必要ならば
```suggestion
INSERT INTO fishes (name,weight,priceInYen) VALUES ("タイ", "1000","1036");
```
として、通貨の交換レートを別テーブルでもつのもいいかもしれません。

Reply:
以下の修正を行いました。
・重さ、値段を単価重量、在庫量へ変更。
・単価重量、在庫量をStirngデータからintデータへ変更。
・もともと作成していた情報テーブルと別に、単位を付けて表示する表示用テーブルを作成。
https://github.com/Moriwaki-9082/Java-10th-topic/pull/6/commits/a59280f478ea43b163d36a0fdeca699d0f4ae44f#diff-4645cfd99037fcae3115a737492ea916503cd68df9d9c43ed42aa041be8c4c27R21-R25

Reply:
okです！